lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3125,11 +3157,12 @@ namespace Js
             return hr;
         }
 
+#if defined(ENABLE_SCRIPT_PROFILING)
         if (attach)
         {
             this->RegisterDebugThunk();
```

Comment:
>RegisterDebugThunk [](start = 18, length = 18)

We use ProfileThunk under debugger and its set in this function, if we are not calling this where are we setting the thunk?